extensions/2.0/Vendor/EXT_feature_metadata/schema/gltf.EXT_feature_metadata.schema.json

```diff
@@ -0,0 +1,43 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "title": "EXT_feature_metadata glTF extension",
+    "type": "object",
+    "description": "glTF extension for applying semantic information to a model through per-vertex and per-texel feature assignment and storage of feature properties.",
+    "allOf": [ { "$ref": "glTFProperty.schema.json" } ],
+    "properties": {
+        "featureClasses": {
```

Comment:
For better interoperability with [`3DTILES_metadata`](https://github.com/CesiumGS/3d-tiles/pull/431) should `featureClasses` be renamed to `classes`?

Same with `featureClasses` being renamed to `class`

Reply:
@lilleyse @sanjeetsuhag 
Just to make sure I understand, the intent here is that in every case, it's a "class", but whether it's a tileset/layer/tile/content/feature "class" depends on what it's scoped to? And in the case of glTF the only possibility here is that it's scoped to a feature.

though then what about the `featureTable` naming, would that have to change too?

Reply:
I changed `featureClass` -> `class` and `featureClasses` -> `classes`